doc/switch-components.md

```diff
@@ -0,0 +1,60 @@
+# Controlled Switch Component
+https://reactjs.org/docs/forms.html#controlled-components
+
+- Our custom switch component substituting reactstrap's <CustomInput type="switch"/>
+- The input’s value is driven by the React state
+- parameters
+    - isOn (boolean) - sets state
+    - toggle (function) - triggered upon click
+    - title (string / optional)
+    - disabled (boolean / optional)
+
+- example:
+<!-- 
```

Comment:
@petrKavulok Wrap code examples into code template as it is e.g. here https://github.com/TeskaLabs/asab-webui/edit/master/doc/about.md 
Dont use markdown comments for it

Reply:
> Also, next time please rebase instead of merge locally, when you have conflicts

sorry about this, i thought i was rebasing, not sure how this happened. I'll be more careful next time